lib/charms/layer/jenkins/configuration.py

```diff
@@ -80,3 +81,7 @@ def _set_prefix(self, prefix):
             return True
 
         return False
+
+    def _get_url_config(self, url):
+        """Get Jenkins Url configuration node"""
+        return "<jenkinsUrl>" + url + "</jenkinsUrl>" if url else ""
```

Comment:
This is fine, but I think it'd be a tad more elegant to use the "if" support in Jinja templates:

http://jinja.pocoo.org/docs/2.9/templates/#if

It's a minor change, but please let me know what you think.